api/proto/disperser/disperser.proto

```diff
@@ -73,12 +73,10 @@ message DisperseBlobRequest {
 	// The data to be dispersed.
 	// The size of data must be <= 2MiB.
 	bytes data = 1;
-	// Security parameters allowing clients to customize the safety (via adversary threshold)
-	// and liveness (via quorum threshold).
-	// Clients can define one SecurityParams per quorum, and specify multiple quorums.
+	// The quorums to which the blob will be sent, in addition to any required quorums. 
```

Comment:
Does this mean the users do not need to specify the required quorums (e.g. if this field is empty, it's valid and will be handled with required quorums)?

And it's fine to specify quorums in here that overlap with required quorums?

Reply:
Since we are removing required quorums from this PR, I will remove this part of the comment.